src/org/ods/orchestration/util/Project.groovy

```diff
@@ -352,40 +379,55 @@ class Project {
-                                    !issue.status.equalsIgnoreCase(JiraDataItem.ISSUE_STATUS_DONE) &&
-                                    !issue.status.equalsIgnoreCase(JiraDataItem.ISSUE_STATUS_CANCELLED)
-                        }
-                        .collect { key, issue ->
-                        return key
-                        }
+                result[type] = data[type].findAll { k, v -> issueIsWIP(v) }.keySet() as List<String>
             }
         }
-
         return result
     }
 
-    protected Map convertJiraDataToJiraDataItems(Map data) {
-        JiraDataItem.TYPES.each { type ->
-            if (data[type] == null) {
-                throw new IllegalArgumentException(
-                        "Error: Jira data does not include references to items of type '${type}'.")
+    /**
+     * Gets the document chapter issues and puts in a format ready to query from levadocumentusecase when retrieving
+     * the sections not done
+     * @param data jira data
+     * @return dict with map documentTypes -> sectionsNotDoneKeys
+     */
+    protected Map<String,List> computeWipDocChapterPerDocument(Map data) {
+        (data[JiraDataItem.TYPE_DOCS] ?: [:])
+            .values()
+            .findAll { issueIsWIP(it) }
+            .collect { chapter ->
+                chapter.documents.collect { [doc: it, key: chapter.key] }
+            }.flatten()
+            .groupBy { it.doc }
+            .collectEntries { doc, issues ->
+                [(doc as String): issues.collect { it.key } as List<String>]
             }
+    }
 
-            data[type] = data[type].collectEntries { key, item ->
-                return [key, new JiraDataItem(item, type)]
-            }
+    protected boolean issueIsWIP(Map issue) {
+        issue.status != null &&
+            !issue.status.equalsIgnoreCase(JiraDataItem.ISSUE_STATUS_DONE) &&
+            !issue.status.equalsIgnoreCase(JiraDataItem.ISSUE_STATUS_CANCELLED)
+    }
+
+    protected Map convertJiraDataToJiraDataItems(Map data) {
+        JiraDataItem.TYPES.each { type ->
+            if (data.containsKey(type)) {
+                data[type] = data[type].collectEntries { key, item ->
+                    [key, new JiraDataItem(item, type)]
+                }
+            } //else {
+                //throw new IllegalArgumentException(
+                //    "Error: Jira data does not include references to items of type '${type}'.")
+            //}
         }
 
         return data
     }
 
     List<JiraDataItem> getAutomatedTests(String componentName = null, List<String> testTypes = []) {
         return this.data.jira.tests.findAll { key, testIssue ->
-            def result = testIssue.status.toLowerCase() == JiraDataItem.ISSUE_STATUS_DONE &&
-                    testIssue.executionType?.toLowerCase() == JiraDataItem.ISSUE_TEST_EXECUTION_TYPE_AUTOMATED
+            def result = testIssue.executionType?.toLowerCase() == JiraDataItem.ISSUE_TEST_EXECUTION_TYPE_AUTOMATED
```

Comment:
This line changes the expectation that only tests are processed which are DONE. Are we fine with this change?

Reply:
This change was done by Marti on 14/08. Now we have all the manual tests passed. I think it should be ok.

Reply:
@s2oBCN it changes the logic which was based on a constraint of mine. I think it's worth having a discussion.